css/installation/demos/vars/style.css

```
@@ -0,0 +1,40 @@
+:root {
+  --main-bg-color: #2196f3;
+
+  --main-text: #fff;
+  --black: #000000;
+
+  --blue-lighten: #64b5f6;
+  --blue-darken: #1565c0;
+
+  --purple: #9c27b0;
+  --purple-light: #4a148c;
+  --purple-depp: #ce93d8;
+
+  --grey: #9e9e9e;
+  --grey-lighten: #e0e0e0;
+  --grey-darken: #424242;
+
+}
+
+html {
+  height: 100%;
+  display: grid;
+  place-items: center;
+}
+
+body {
+  font-family: 'Roboto';
+  background-color: var(--main-bg-color);
+}
+
+h1 {
+  color: var(--main-text);
+  font-size: 50px;
+}
+
+p {
+  color: var(--grey-darken);
+  font-size: 24px;
+  font-weight: bold;
```

```suggestion
  font-size: 18px;
```

Жирный шрифт для параграфов лучше не надо.